docs/Telemetry.md

```diff
@@ -0,0 +1,53 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+
+## Telemetry
+
+Accessibility Insights for Windows uses telemetry to better understand what features are most helpful to users, as well as to help identify potential issues that users are experiencing.
+
+### Reporting a telemetry action
+Telemetry actions typically indicate a user-initiated action (starting the app, clicking a button, etc.). They are also used to report the result of a user action (for examplle, a user attempt to open a file failed because the file was in an unrecognized format). At this time, telemetry actions are *not* supported in extensions. The telemetry classes are defined in the `AccessibilityInsights.Desktop.Telemetry` namespace. The `Logger` class provides 2 variants of the static `Logger.PublishTelemetryEvent` method:
+
+#### Single-property action
+This version reports an action with a single associated property:
+```
+Logger.PublishTelemetryEvent(
+    TelemetryAction.Mainwindow_Timer_Started,       // Name of action 
+    TelemetryProperty.Seconds,                      // Name of property
+    count.ToString(CultureInfo.InvariantCulture));  // Value of property (must be a locale-agnostic string)
+```
+
+#### Multi-property action
+This version reports an action with multiple associated properties:
+```
+Logger.PublishTelemetryEvent(TelemetryAction.Bug_Save, new Dictionary<TelemetryProperty, string>
+    {
+        { TelemetryProperty.RuleId, ruleId },  // values are locale-agnostic strings
+        { TelemetryProperty.Url, url},
+    });
+```
+
+The `TelemetryAction` and `TelemetryProperty` enumerations can be extended to add new actions or properties.
+
+#### Context Properties
```

Comment:
>Context Properties [](start = 5, length = 18)

The logger implementation of context properties isn't defined here. As such, I'd suggest putting them as a separate H3 (probably after exceptions), as opposed to an H3 inside actions. The AIWin implementation of telemetry applies context properties to both actions and exceptions